src/store/general/state.ts

```diff
@@ -111,6 +112,7 @@ function state(): GeneralStateInterface {
     currentBlock: 0,
     activeInflationConfiguration: undefined,
     inflationParameters: undefined,
+    blockTimeInSeconds: 12,
```

Comment:
This is default?
Can it be uninitialised until fetched from api?

Reply:
It can :)